### PR TITLE
fix: screener should not be able edit their own schedule

### PR DESCRIPTION
**Pull request by jas2996**
https://certushealth.atlassian.net/browse/CM-38

### CODE REVIEW

File: emr/build/js/calendar1.js [hunk 1]
```diff
@@ -26,6 +26,12 @@ function get_data() {
             type: 'POST',
             dataType: 'json',
             success: function(data) {
+                var accountType = ""
+                if (Object.getOwnPropertyNames(data)[0] == "scheduleView") {
+                    accountType = "admin"
```
Comment (sandeepsuri) [inline emr/build/js/calendar1.js:31]: should there also be an accountType for hospital admin as well?

**Review by sandeepsuri — COMMENTED** [review]
should there also be an accountType for hospital admin as well?

File: emr/build/js/calendar1.js (hunk 1 above)
Comment (jas2996) [reply emr/build/js/calendar1.js:31]: No because `scheduleView` is for both admin and hospital admin

Comment (sandeepsuri) [reply emr/build/js/calendar1.js:31]: Okay, in that case it's good with me

**Comment by sandeepsuri** [comment]
The code looks good to me!

**Comment by jas2996** [comment]
Great!

**Pull request merged**